website/src/pages/extensions/index.tsx

```
@@ -1,24 +1,679 @@
+    // Sort extensions
+    filtered.sort((a, b) => {
+      switch (sortBy) {
+        case 'name':
+          return a.name.localeCompare(b.name);
+        case 'date':
+          return new Date(b.lastUpdated).getTime() - new Date(a.lastUpdated).getTime();
+        case 'rating':
+          return b.rating - a.rating;
+        case 'popularity':
+        default: {
+          const aScore = a.rating * (parseInt(a.installs.replace(/[^\d]/g, '')) || 0);
```

Biome (2.1.2)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)